test/testlib/nuodb_admin_utilities.go

```diff
@@ -142,6 +143,18 @@ func StartAdminTemplate(t *testing.T, options *helm.Options, replicaCount int, n
 		}
 	}
 
+	admin0 := fmt.Sprintf("%s-nuodb-cluster0-0", helmChartReleaseName)
+
+	RunOnNuoDBVersionCondition(t, ">=6.0.0", func(version *semver.Version) {
```

Comment:
Nit: The `RunOnNuoDBVersionCondition` will be called for every test now.  It creates a pod to check the NuoDB version which takes some time. You could cache the fetched version in a global variable to minimize the performance impact.